src/BenchmarksServer/Startup.cs

```diff
@@ -4082,6 +4087,107 @@ private static async Task UseMonoRuntimeAsync(string runtimeVersion, string outp
             }
         }
 
+        private static async Task AOT4Mono(string dotnetSdkVersion, string runtimeVersion, string outputFolder)
+        {
+            try
+            {
+                //Download dotnet sdk package
+                var dotnetMonoPath = Path.Combine(_rootTempDir, "dotnet-mono", $"dotnet-sdk-{dotnetSdkVersion}-linux-x64.tar.gz");
+                if (Directory.Exists(Path.GetDirectoryName(dotnetMonoPath)))
+                {
+                    Directory.Delete(Path.GetDirectoryName(dotnetMonoPath));
+                }
+                Directory.CreateDirectory(Path.GetDirectoryName(dotnetMonoPath));
+
+                Log.WriteLine($"Downloading dotnet skd package for mono AOT");
+
+                var found = false;
+                var url = $"https://dotnetcli.azureedge.net/dotnet/Sdk/{dotnetSdkVersion}/dotnet-sdk-{dotnetSdkVersion}-linux-x64.tar.gz";
```

Comment:
Isn't the sdk already downloaded and installed by a previous "standard" step?

Reply:
This is a local dotnet that I switch the runtime to mono, so that I could use it for AOT. I don't want to do that to the installed dotnet which is shared by CoreCLR and Mono.